components/DropdownLight/DropdownLight.jsx

```diff
@@ -41,8 +48,16 @@ const ArrowIcon = styled(Icon)`
   width: 24px;
 `;
 
-const Wrapper = styled.div`
-  position: relative;
+const ComponentWrapper = styled.div`
```

Comment:
why `ComponentWrapper` instead `Wrapper`?

Reply:
I thought it could be more objective, I followed clean code standards.